src/components/Messages/Message/index.js

```diff
@@ -36,11 +37,32 @@ import {
 import styles from './styles.scss';
 
 const onClickFeedback = () => {
-	const { postChatUrl } = store.state;
+	const { postChatUrl, room: { _id: rid } = {} } = store.state;
+
+	const loggerPayload = {
+		room_id: rid,
+		category: 'Survey',
+		action: 'link_clicked',
+		properties: {},
+		event_type: 'customer_action',
+	};
+	Livechat.sendLogsToSns(loggerPayload);
+
```

Comment:
How about we maintain all kinds of log payloads in some enum or something in one file and use it everywhere? And also may be creating a generator function that only takes rid and enum and returns the expected payload. So that it looks nice and clean and short 😁